appveyor.yml

```diff
@@ -126,7 +127,7 @@ build:
 
 # scripts to run after tests
 after_test: 
-    - .\packages\OpenCover.4.6.166\tools\OpenCover.Console.exe -register:user -target:"C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\IDE\MSTest.exe" -targetargs:"/noresults /noisolation /testcontainer:"".\GerbilTest\bin\Debug\GerbilTest.dll" -filter:"" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -output:.\MyProject_coverage.xml
+    - .\packages\OpenCover.4.6.166\tools\OpenCover.Console.exe -register:user -target:"C:\projects\myproject\packages\NUnit.Runners.2.6.4\tools\nunit-console.exe .\GerbilTest\bin\Debug\GerbilTest.dll" -targetargs:"/noshadow" -filter:"" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -output:.\MyProject_coverage.xml
```

Comment:
This is where AppVeyor vomits up an error message about an invalid target. URI checks out, something is happening with Nuget packages before this stage.